app/main.py

```diff
@@ -1,3 +1,88 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from datetime import datetime
+
+from app.customer import Customer
+from app.car import Car
+from app.shop import Shop
+from app.go_to_shop import count_trip_to_shop_value
+
+
+def shop_trip() -> None:
```

Comment:
80-line function is bad for reading
separate your function into a few functions with own logic